doc/users/prev_whats_new/whats_new_3.6.0.rst

```diff
@@ -58,6 +58,7 @@ With ``layout='tight'`` or ``'constrained'``, Axes with a fixed aspect ratio
 can leave large gaps between each other:
 
 .. plot::
+    :alt: A 5 by 3 inch figure with subplots displayed in 2 rows and 2 columns; Subplots have large gaps between each other.
```

Comment:
Suggestion: instead of mentioning the size of the image, which I'm not sure is relevant for comprehension, we could mention that it is labeled `fixed-aspect plots, layout='constrained'` as I don't think this can be inferred from the surrounding text.

Reply:
Should the title be in quotes, or without quotes? I a seeing other examples of alt text describing text inside an image with quotes like this: https://github.com/MarsBarLee/blog.scientific-python.org/pull/1/files#diff-654f2c319a83b43081d6ee17595fd9bce8f36cfc4f29e59efc0a98563cc37edcL76 (line 105)

Reply:
In this case, in my opinion it may make sense to use quotes as the text is the result of a command from matplotlib. The quotes would then help a reader understand that better.